core-data/actions.js

```diff
@@ -71,6 +71,20 @@ export function receiveEntityRecords( kind, name, records ) {
 	};
 }
 
+/**
+ * Returns an action object used in signalling that taxonomies have been received.
+ *
+ * @param {Array} taxonomies Taxonomies received.
+ *
+ * @return {Object} Action object.
+ */
+export function receiveTaxonomies( taxonomies ) {
+	return {
+		type: 'RECEIVE_TAXONOMIES',
+		taxonomies: taxonomies,
```

Comment:
This could just be `taxonomies`, instead of `taxonomies: taxonomies`.

Reply:
9a96bed94e2f07b53ac409e0a3d01acf989c4e19